index.html

```diff
@@ -0,0 +1,279 @@
+                <div class="feature__container_side">
+                  <p class="category__portfolio">CANOPY</p>  
+                </div>
+                  <div class="canopy__design"></div>
+                  <div class="canopy__layout">
+                   <p class="canopy__backend">Back End Dev</p>
+                   
+                  </div>
+                  <div class="backend__design">
+                  <p class="category__port">2015</p>
+                  </div>
+              </div>
+              <p class="navigation__paragraph">
+                A daily selection of privately personalized reads; no accounts or
+                sign-ups required</p>
+              <div>
+                <ul class="skills">
+                  <li class="skills__container">
+                    <p>HTML</p>
+                  </li>
+                  <li class="skills__container">
+                    <p>CSS</p>
+                  </li>
+                  <li class="skills__container">
+                    <p>JavaScript</p><br/>
+                  </li>
+                </ul>
+              <div class="Portfolio__project__layout">
+                <div class="Portfolio__project__design">
+                  <button class="Portfolio__project__content">See Project </button>
+                </div>
+              </div>
+            </div>
+            </div>
+            </div>
+            <div class="navigation__work">
+              <img src="img/work/port2.png" class="portfolio_f_image" alt="" />
+              <div class="portfolio__img__content">
+              <h3 class="portfolio__tonic">Multi-Post Stories</h3>
+              <div class="feature__container">
+                <div class="feature__container_side">
+                  <p class="category__portfolio">CANOPY</p>  
+                </div>
+                  <div class="canopy__design"></div>
+                  <div class="canopy__layout">
+                   <p class="canopy__backend">Back End Dev</p>
+                   
+                  </div>
+                  <div class="backend__design">
+                  <p class="category__port">2015</p>
+                  </div>
+              </div>
+              <p class="navigation__paragraph">
+                A daily selection of privately personalized reads; no accounts or
+                sign-ups required</p>
+              <div>
+                <ul class="skills">
+                  <li class="skills__container">
+                    <p>HTML</p>
+                  </li>
+                  <li class="skills__container">
+                    <p>CSS</p>
+                  </li>
+                  <li class="skills__container">
+                    <p>JavaScript</p><br/>
+                  </li>
+                </ul>
+              <div class="Portfolio__project__layout">
+                <div class="Portfolio__project__design">
+                  <button class="Portfolio__project__content">See Project </button>
```

Comment:
- [ ] Great job adding the button element for the see project buttons. But this came with a new UI issue. Kindly style the buttons to match the Figma template. You can remove the background and the border of the button and make it fill the div.
### Your design
![image](https://github.com/AsciencioAlex/Mobile-Finish-Portfolio/assets/94551484/d58b0ec2-563f-48ab-a188-db778f19141d)
### Figma 
![image](https://github.com/AsciencioAlex/Mobile-Finish-Portfolio/assets/94551484/7604c268-80f1-43f7-b330-3fb395a26944)